Progress bar.py

```diff
@@ -0,0 +1,45 @@
+import math, colorama
```

Comment:
I think this is more deployment ready to a public repository.
Here, is the code
```
import math
import colorama
from colorama import Fore, Style

colorama.init()

def progress_bar(progress, total, color=Fore.CYAN, complete_color=Fore.GREEN):
    percent = 100 * (progress / total)
    bar_length = 50
    completed_length = int(bar_length * (progress / total))
    bar = '█' * completed_length + '-' * (bar_length - completed_length)

    print(f'\r|{color}{bar}{Style.RESET_ALL}| {color}{percent:.2f}%{Style.RESET_ALL}', end='', flush=True)

    if progress == total:
        print(f'\r|{complete_color}{bar}{Style.RESET_ALL}| {complete_color}{percent:.2f}%{Style.RESET_ALL}')

numbers = [x * 5 for x in range(2000, 3000)]
lenght = len(numbers)

progress_bar(0, total=lenght)

results = []
for i, x in enumerate(numbers):
    results.append(math.factorial(x))
    progress_bar(i + 1, total=lenght)

print("\nFactorials calculated successfully!")

```

Reply:
See, this, add this to your code.
Document it more properly if want, 
then recommit it.
It will be all good.